packages/block-editor/src/components/global-styles/advanced-panel.js

```diff
@@ -28,6 +28,10 @@ export default function AdvancedPanel( {
 			css: newValue,
 		} );
 		if ( cssError ) {
+			// Pass a wrapping selector to transformStyles to ensure that the CSS
+			// tested resembles actual CSS rules. This is necessary because the additional CSS
+			// field supports CSS rules that are not wrapped in a selector.
```

Comment:
To be precise, we need to pass a wrapper selector because otherwise the transformStyle function will [return before it does the error checking](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/utils/transform-styles/index.js#L18). I don't think it really matters whether the actual CSS is a whole rule or just a property or two. Afaict inputting a single property will output the same (unwrapped) property, whether we pass a wrapper or not.

It would probably be better to have a dedicated error checking function instead of using this one 🤔  That might be a good improvement to add to the list for additional CSS!